dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/ResourcesService.java

```diff
@@ -644,18 +654,20 @@ public Result updateResourceContent(int resourceId, String content) {
         if (StringUtils.isNotEmpty(resourceViewSuffixs)) {
             List<String> strList = Arrays.asList(resourceViewSuffixs.split(","));
             if (!strList.contains(nameSuffix)) {
-                logger.error("resouce suffix {} not support updateProcessInstance,  resource id {}", nameSuffix, resourceId);
+                logger.error("resource suffix {} not support updateProcessInstance,  resource id {}", nameSuffix, resourceId);
                 putMsg(result, Status.RESOURCE_SUFFIX_NOT_SUPPORT_VIEW);
                 return result;
             }
         }
 
+        String tenantCode = getTenantCode(resource.getUserId(),result);
+        if (StringUtils.isEmpty(tenantCode)){
```

Comment:
Is necessary to putMsg and log error?

Reply:
Since a tenant is required to upload a resource, the resource and tenant are associated through the user. If the user on resouce is illegal. Then the tenant cannot be obtained, and the null pointer exception will be thrown when the tenantCode is obtained.

由于上传resource需要tenant，resource和tenant是通过用户关联，如果resource上的用户是非法的。那么就获取不到tenant了， 获取tenantCode也就会抛空指针异常

Reply:
add error log  in getTenantCode method